### ISSUE TITLE
Feature Request: Expose photo date folders to jotta-cli 

### ISSUE THREAD

**Issue by mabahj** [jotta/jotta-cli-issues#128]
Make sure you are running the latest version of jotta-cli before reporting an issue.

**jotta-cli release (`jotta-cli version`):**
```
-------------------------------------------
jottad executable : /usr/bin/jottad
jottad appdata    : /var/lib/jottad
jottad version    : 0.8.36922
jotta-cli version : 0.8.36922
-------------------------------------------
```

**Description of problem:**
Related to #125. The jotta-cli only exposes "Albums" and "Location Albums". Some photos are therefore difficult (or impossible) to find:
* Not all photos have location tags
* One may (for example during a rainy December) want to download all pictures from 2020/June and 2020/July

**Expected:**
```
%> jotta-cli ls Photos
Albums Location Albums Date Albums          # Note - the space separation here (and when listing photos) makes the output a bit tricky to both read and process
%> jotta-cli ls Photos/Date Albums/
2017 2018 2019 2020
%> jotta-cli ls Photos/Date Albums/2020/
January May June July                                   # (or 01 02 03 etc)
%> jotta-cli ls Photos/Date Albums/2020/July 
15072009258.jpg 15072009258.jpg 15072009259.jpg
%> jotta-cli download Photos/Date Album/2020/July/* .
(Downloading all from July 2020)
%> jotta-cli download Photos/Date Album/2020/July/15072009259.jpg .
(Downloading this photo)
```

**jotta-cli status (`jotta-cli status`):**
```
N/A
```

**Relevant logs for the issue (`cat ~/.jottad/jottabackup.log`)**
```
N/A
```

**Traceback**
```
N/A
```

**Additional info:**

**Comment by Kimbsen** [comment]
We are very close to releasing almost exactly this. Will keep you posted!

**Comment by Kimbsen** [comment]
We just released 0.9.38023. 

It adds a new `ls` and `download` target Photos/Timeline you can use to list and download photos from your timeline!

**Comment by mabahj** [comment]
Beautiful! Thank you!
